fsspec/registry.py

```diff
@@ -202,6 +202,10 @@ def register_implementation(name, cls, clobber=True, errtxt=None):
         "class": "webdav4.fsspec.WebdavFileSystem",
         "err": "Install webdav4 to access WebDAV",
     },
+    "dvc": {
+        "class": "dvc.api.DVCFileSystem",
+        "err": "Install dvc to access DVCFileSystem",
+    },
```

Comment:
Let's also add to https://github.com/fsspec/filesystem_spec/blob/master/docs/source/api.rst#other-known-implementations

Reply:
Should we add a link to the docs or a repository?

Reply:
No, the name of the package should be enough, people can find it from the pypi page.

Reply:
Looks like this comment was resolved by mistake?

Reply:
Sorry, I mean: please don't change this text, it's good as it is. Indeed adding to the docs is a good idea, though.

Reply:
@skshetry Btw, is this going to be operational in open/read calls? Just trying to remember if the urls will get handled correctly, since dvcfs wasn't present in the registry before.

Reply:
For the record: @skshetry explained that it works with additional kwargs passed through like this:
```
with open("dvc://model.pkl", url="https://github.com/iterative/example-get-started.git", rev="main") as f:
   d = pickle.load(f)
```